webofneeds/won-owner-webapp/src/main/webapp/app/reducers/need-reducer.js

```diff
@@ -0,0 +1,1241 @@
+/**
```

Comment:
isnt this the old file? should be removed right?

Reply:
Ah, that was a rebase fuck up. It's fixed now :)